tools/independent/RuntimeGraphGenerator/Program.cs

```diff
@@ -0,0 +1,105 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
```

Comment:
Are these all "dupes" of the existing RuntimeGraphGenerator?  Can these files be moved instead?